src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/styles.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {Color} from 'app/utils/theme';
+import space from 'app/styles/space';
+
+// TODO(style): the color #fffcfb and  #e7c0bc are not yet in theme and no similar theme's color was found.
+const BreadCrumb = styled('li')<{hasError?: boolean}>`
+  font-size: ${p => p.theme.fontSizeMedium};
+  position: relative;
+  padding: ${space(1)} ${space(3)} ${space(0.75)} ${space(3)} !important;
+  display: grid;
+  grid-template-columns: 26px 1fr 50px;
+  grid-gap: ${space(1.5)};
+  :before {
+    content: '';
+    display: block;
+    width: 2px;
+    top: 0;
+    bottom: 0;
+    left: 32px;
+    background: ${p => p.theme.borderLight};
+    position: absolute;
+  }
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  :last-child:before {
+    bottom: calc(100% - ${space(1)});
+  }
+  ${p =>
+    p.hasError &&
+    css`
```

Comment:
Do you need `css` given you are in a template already.

Reply:
Actually, I don't need the `css`, but I am using it to have a good indentation